### PR TITLE
Adding About

### PR DESCRIPTION
**Pull request by lxwyndxl**
to: @jessicatai Adding the About section
screen shot 2016-10-09 at 2 43 19 am

### CODE REVIEW

File: src/app/components/about.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import '../../stylesheets/components/about.css';
```
Comment (jessicatai) [inline src/app/components/about.js]: try w/o the .css

File: src/app/components/about.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      </h2>
+
+      <h1 className="name">
+        {firstName} <span className="last-name">{lastName}</span>
+      </h1>
+
+      <p className="passage-text">
+        {
+          text.map((paragraph, index) => {
+            return (<span key={index} className="passage-paragraph">{paragraph}</span>);
```
Comment (jessicatai) [inline src/app/components/about.js]: nit: you can self close `<span ... />`

File: src/stylesheets/components/about.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  font-size: 35px;
+  font-weight: 700;
+  text-align: center;
+  text-transform: uppercase;
+  margin-top: 10px;
+  color: #333;
+}
+
+.groom .last-name {
+  color: #1fbec3;
```
Comment (jessicatai) [inline src/stylesheets/components/about.css]: make colors constant for reuse

File: src/stylesheets/components/hero.css [hunk 4]
```diff
@@ -48,26 +49,31 @@
...
+}
+
+.welcome-text-2 {
+  font-size: 60px;
 }
 
-.welcome-text-2,
 .welcome-text-3 {
-  font-size: 62px;
+  font-size: 61px;
```
Comment (jessicatai) [inline src/stylesheets/components/hero.css]: can we just use 60 for both or 61 for both

File: src/stylesheets/components/hero.css [hunk 5]
```diff
@@ -48,26 +49,31 @@
...
-  font-size: 103px;
+  font-size: 101px;
 }
 
 .wedding-date {
   width: 400px;
   color: #333;
-  text-align: justify;
+  text-shadow: 0px 1px 1px #fff;
+  text-align: center;
```
Comment (jessicatai) [inline src/stylesheets/components/hero.css]: could make a shared class so you can DRY this up

File: src/app/components/about.js (hunk 1 above)
Comment (lxwyndxl) [inline src/app/components/about.js]: no go

**Pull request merged**